website/style_guide.md

```diff
@@ -230,7 +230,15 @@ For example:
 
 Do not document function arguments unless they are non-obvious of their intent
 (though if they are non-obvious intent, the API should be considered anyways).
-Therefore `@param` should generally not be used.
+Therefore `@param` should generally not be used. If `@param` is used, it should
+not include the `type` as Typescript is already strong typed.
```

Comment:
It should be 'already strong_ly_ typed.'

Reply:
nailed it